yarn.lock

```diff
@@ -64,6 +64,27 @@
     json5 "^2.1.2"
     semver "^6.3.0"
 
+"@babel/core@^7.17.7":
```

Comment:
Should this changes be added? Im seeing that is on root level?

Reply:
I don't figured how those changes came up. I removed them now, on changes.